extensions/amp-social-share/1.0/test/test-amp-social-share.js

```diff
@@ -226,6 +229,49 @@ describes.realWin(
       ).to.be.equal('inherit');
     });
 
+    it('should show default and custom focus indicator styles', async () => {
```

Comment:
@jridgewell @samouri Was hoping to get a second set of eyes here.  I noticed that with this test that I've added, it only passes the first time when I run it locally.

So I run it with the `--watch` flag, so that each time I make a change it reruns, but this causes it to fail after the first run.

Specifically below at line 251, the computed styles are not applied in the second run and the test complains that I should receive `rgb(255, 255, 255) none 0px`.

Wanted to check in and see if there might be some type of runtime related flag I should be checking for?  Or if there's something else I am missing.